message-router/message-router.yml

```diff
@@ -1,5 +1,5 @@
-tagline: Route message to different output channels
-desc: A message router consumes a message and republishes it to a different message channel depending on a set of conditions.
-helps: Ballerina enables routing messages to different endpoints by reading the message and using basic control structures such as if and switch statements.
-tags: ["Message Router", "Message Channnel", "Message Endpoint"]
+tagline: Route messages to different output channels
+desc: Message router consumes a message from one channel and republishes it to a different channel depending on a set of conditions.
+helps: "Ballerina supports conditional logic with if-else and match statements. This can be used to route messages based on message content, header, or any custom logic. Ballerina type system supports union types (e.g.: Country type below) which helps to define choices in a type-safe and readable manner." 
```

Comment:
Why do we need quotation marks here since they're not used elsewhere?

Reply:
I added it since vscode showed red without it. didn't look into the reason.

Reply:
Looks like it's due to the `:`. removed it.